src/main/java/tutorial/Jail.java

```diff
@@ -0,0 +1,62 @@
+package tutorial;
+// How to go to Jail:
+// roll double thrice consecutively
+// go to jail chance card (if passed go to land on card chest, collect go money, else go directly to jail)
+// land on jail (visiting, no penalty)
+
+import java.util.ArrayList;
+
+public class Jail {
+    Player player;
+    private int turnsInJail;
+    private boolean inJail;

```

Comment:
Should the inJail Variable be public? - so that it can be checked in other classes if a player is in jail

Reply:
there is a getter method for inJail.